helm/kubernetes-cluster-prep/templates/tests/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+# Run from dir in which this script resides
+cd "$(dirname "$0")"
+
+TAG=latest
+
+echo "================================================"
+echo "     Building Cluster Prep Helm test image"
+echo "================================================"
+docker build -t "rpothier/conjur-cluster-prep-test:$TAG" .
```

Comment:
Should be `cyberark/conjur-cluster-prep-test:$TAG`

Reply:
same as above